sklearn/discriminant_analysis.py

```diff
@@ -36,16 +36,17 @@ def _cov(X, shrinkage=None, covariance_estimator=None, standardize=False):
     X : array-like, shape (n_samples, n_features)
         Input data.
 
-    covariance_estimator : estimator, or None, default=None
+    covariance_estimator : estimator, default=None
         The covariance estimator is a BaseEstimator with a fit
-        method and a .covariance_ attribute like the estimators in
-        sklearn.covariance
+        method and a ``covariance_`` attribute like the estimators in
+        `sklearn.covariance`
 
         if None the shrinkage parameter drives the estimate.
 
     standardize : bool
-        if True standardize the data before applying the covariance estimator
-        Note: only used when covariance_estimator is not None
+         if ``True`` standardize the data before
```

Comment:
I think you have an extra space at the beginning of these two lines. Also, all new sentences here should start with a capital letter.